packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -202,6 +219,32 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
                 return detectTruncation
                     ? this.state.isTruncated
                     : truncateLength > 0 && content.length > truncateLength;
+            case TruncatedPopoverMode.WHEN_TRUNCATED_APPROX:
+                if (!detectTruncation) {
+                    return truncateLength > 0 && content.length > truncateLength;
+                }
+                const {
+                    getApproximateCharWidth: approxCharWidth,
+                    getApproximateLineHeight: approxLineHeight,
+                    getCellHorizontalPadding: horizontalPadding,
+                    getNumBufferLines: numBufferLines,
+                } = TruncatedFormat.resizeRowsByApproximateHeightDefaults;
+
+                const cellWidth = parseInt(this.props.parentCellWidth, 10);
```

Comment:
`parentCellWidth` is an optional prop. Can you add a null check before invoking `parseInt`?

Reply:
Also, why is this prop a string in the first place?

Reply:
It's only optional because the end-user isn't expected to use it -- the thing gets cloned and the value injected whenever it's used in the right spot inside the table. In fact any values end-users put there will get overwritten. I can still put the null checks, though. 

It's a string because it's not "10" it's "10px", because it comes from styles. And we didn't care about filtering it down above, because it was about triggering changes.

Reply:
...do we want to do the parsing at the cell level instead of here? I think that's probably slightly better, will-do.